include/zmq.h

```diff
@@ -101,6 +101,9 @@ typedef unsigned __int8 uint8_t;
 #include <stdint.h>
 #endif
 
+// needed for sigset_t definition
+#include <signal.h>
```

Comment:
This should be ifdefd for windows too just in case

Reply:
Will do.